src/components/Discussion/Composer/docs.imports.js

```diff
@@ -66,6 +66,12 @@ export const CommentComposerPlayground = () => {
     actions: {
       openDiscussionPreferences: () => Promise.resolve({ ok: true })
     },
+    getLabel: text => {
+      if (text.indexOf('*') > -1 && text.indexOf('\\*') === -1) {
+        return t('styleguide/CommentComposer/formattingHelp')
```

Comment:
Maybe go for a specific key name, what it is used for, like:

```suggestion
        return t('styleguide/CommentComposer/formatting/asterisk')
```